README.md

```diff
@@ -1 +1,43 @@
-# Portfolio
\ No newline at end of file
+# Project Name
```

Comment:
Thank you so much, I have updated the changes you mention, but I have a question. How can I resolve the conflict in my README file? 

Also, can I merge my work, or it will be from your end?